controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/screwdriver/ScrewdriverApiHandler.java

```diff
@@ -112,20 +113,15 @@ private HttpResponse vespaVersion() {
         return new SlimeJsonResponse(slime);
     }
 
-    private HttpResponse buildJobs(Map<JobType, List<DeploymentTrigger.Job>> jobLists) {
+    private HttpResponse buildJobs(Map<JobType, ? extends List<? extends BuildService.BuildJob>> jobLists) {
```

Comment:
Why is the overly generic type needed?

Reply:
It's that, or
    @SuppressWarnings("unchecked")
    public Map<JobType, List<BuildJob>> jobsToRun() {
        return (Map<JobType, List<BuildJob>>) ((Map<JobType, ? extends List<BuildJob>>) computeReadyJobs().collect(groupingBy(Job::jobType)));
    }
```java

```